jobs/env/ci-kubernetes-e2e-scalability-common.env

```diff
@@ -8,13 +8,12 @@ LOGROTATE_MAX_SIZE=5G
 
 # Use IP-aliases for scalability tests.
 KUBE_GCE_ENABLE_IP_ALIASES=true
-# Turn on profiling for various components and
-# increase throughput in master components.
+# Turn on profiling for various components.
 ETCD_EXTRA_ARGS=--enable-pprof
-CONTROLLER_MANAGER_TEST_ARGS=--profiling --kube-api-qps=100 --kube-api-burst=100
+CONTROLLER_MANAGER_TEST_ARGS=--profiling
```

Comment:
What's the reason for changing this defaults? This is used by almost all our scalability jobs I think.